qemu.go

```diff
@@ -468,13 +470,16 @@ func (q *qemu) setCPUResources(podConfig PodConfig) ciaoQemu.SMP {
 	return smp
 }
 
-func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
+func (q *qemu) setMemoryResources(podConfig PodConfig) (ciaoQemu.Memory, error) {
+	hostMem, err := getHostMemorySize()
+	if err != nil {
+		return ciaoQemu.Memory{}, fmt.Errorf("Unable to read memory info: %s", err)
+	}
+	// add 1G memory space for nvdimm device (vm guest image)
+	memMax := fmt.Sprintf("%dM", int(float64(hostMem)/1024/1024)+1024)
```

Comment:
in the spirit of @jodh-intel , it might be nice if the variable names reflected the units they were holding, maybe like `memMaxMegs` or similar. not a blocker :-)